test-packages/static-app/ember-cli-build.js

```diff
@@ -21,6 +21,12 @@ module.exports = function (defaults) {
     staticAddonTrees: true,
     staticComponents: true,
     staticHelpers: true,
+    skipBabel: [
+      {
+        package: 'qunit',
+        semver: '*',
+      },
+    ],
```

Comment:
This is unfortunate, but at the moment Babel'ing QUnit results in `define` invocation here:

https://github.com/qunitjs/qunit/blob/f3d7f93f0ce23541559bace7d5d3af5a3f894ccc/src/export.js#L38-L41

And since our loader.js doesn't support anonymous AMD, it throws an error.